Tasks/MSBuildV1/MSBuild.ps1

```diff
@@ -24,6 +26,16 @@ try {
     [string]$msBuildVersion = Get-VstsInput -Name MSBuildVersion
     [string]$msBuildArchitecture = Get-VstsInput -Name MSBuildArchitecture
 
+    $msbuildTelemetry = [PSCustomObject]@{
+        MSBuildVersion = $msBuildVersion
+        MSBuildArguments = $msBuildArguments
+        MSBuildLocation = $msBuildLocation
```

Comment:
Paths could contain PII data - I don't think we need to include them to the telemetry.

Reply:
removed this field from telemetry